### PR TITLE
Update impi and addlicense

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,7 +6,6 @@ require (
 	contrib.go.opencensus.io/exporter/jaeger v0.1.1-0.20190430175949-e8b55949d948
 	contrib.go.opencensus.io/exporter/ocagent v0.6.0
 	contrib.go.opencensus.io/exporter/prometheus v0.1.0
-	contrib.go.opencensus.io/resource v0.1.2
```
Comment (james-bebbington) [inline go.mod:9]: That doesn't look related

Comment (bogdandrutu) [reply go.mod:9]: I swear that I did just run `go get package` and `go mod tidy`. Will redo the PR.

Comment (bogdandrutu) [reply go.mod:9]: It may be that one of the dependency had a dependency on `contrib.go.opencensus.io/resource` and go to resolve different versions between transient dependencies added it here. In code we don't directly use that.

Comment (james-bebbington) [reply go.mod:9]: Yea it was probably removed in another PR and `go mod tidy` just wasn't run

Comment (bogdandrutu) [reply go.mod:9]: Me neither :))
